src/components/PokeCards.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable max-len */
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import fetchPokemonData from '../redux/actions/pokeActions';
+import {
+  selectCards,
+  selectLoading,
+  selectError,
+} from '../redux/selectors/pokeSelectors';
+import PokeSearch from './PokeSearch';
+
+const PokeCards = () => {
+  const dispatch = useDispatch();
+  const cards = useSelector(selectCards);
+  const loading = useSelector(selectLoading);
+  const error = useSelector(selectError);
+  const [searchTerm, setSearchTerm] = useState('');
+
+  useEffect(() => {
+    dispatch(fetchPokemonData());
+  }, [dispatch]);
+
+  if (loading) {
+    return <div>Loading...</div>;
+  }
+
+  if (error) {
+    return (
+      <div>
+        Error:
+        {' '}
+        {error}
+      </div>
+    );
+  }
+
+  const pokeList = cards.filter((card) => card.name.toLowerCase().includes(searchTerm.toLowerCase()));
+
+  return (
+    <div className="pokemon-list">
+      <div className="pokemon-list__total">
+        Total Pokemons:
+        {' '}
+        {pokeList.length}
+      </div>
+      <PokeSearch searchTerm={searchTerm} onSearch={setSearchTerm} />
+      <div className="pokemon-list__cards">
+        {pokeList.map((card) => (
+          <div key={card.url}>
+            <Link to={`/details/${card.name}`}>
+              <button type="button">Details</button>
+            </Link>
```

Comment:
- [ ] On the details page, your design is missing the back button. Based on project requirement, When the user clicks on the "Back" button (<), the user navigates to the home page. 
- I can see that the only option to go back now is through the logo. for better user experience and making your webapp more accessible, kindly add a back (<) button on the top left corner of the app just as in the template given.

### Sample missing feature:
![image](https://github.com/Jrmcuervo/react-capstone-metrics-webapp/assets/50990925/67e57fab-52e8-4af7-9d8a-4f97b40fc8a3)

### Your details page:
![image](https://github.com/Jrmcuervo/react-capstone-metrics-webapp/assets/50990925/bf3d9e77-8317-4679-baa1-a908b7eb8ac2)